src/util/debug_module.js

```diff
@@ -501,9 +501,13 @@ function log_event(event) {
         return;
     }
     event.pid = process.pid;
-    if (syslog) {
+    if (syslog && config.LOG_TO_SYSLOG_ENABLED) {
         syslog(config.EVENT_LEVEL, JSON.stringify(event), config.EVENT_FACILITY);
     }
+    if (console_wrapper && config.LOG_TO_STDERR_ENABLED) {
+        const formatted_event = int_dbg.message_format("EVENT", [JSON.stringify(event)]);
+        process.stderr.write(formatted_event.message_console + '\n');
```

Comment:
we re-thinking about it, should we keep the exact same format of event in rsyslog for MAPS team?

Reply:
yep